architecture/core_concepts/routes.adoc

```diff
@@ -621,6 +629,30 @@ the destination. This field is required, but only for re-encryption.
 [[router-sharding]]
 == Router Sharding
 
+Sharding allows the operator to define multiple router groups. Each
+router in the group serves only a subset of traffic (sharding).
+
+When routers are sharded, a given route is bound to a router in the group. The route
+binding ensures uniqueness of the route across the shard.  Uniqueness allows secure
+and non-secure versions of the same route to exist within a single shard.
+This implies that routes now have a visible life cycle that moves from created to
+bound to active.
+
+In the sharded environment the first route to hit the shard reserves the right to
+exist there indefinitely. Even across restarts.
+
+During a green/blue deployment a route may exist multiple times.  An OpenShift
```

Comment:
Yeah, but that requires two routers... one for the old, and one from the new.  And this paragraph doesn't cover it.

Of course, Rajat is working on the A/B bleedover which will do this for a cutover when the services that are provided are identical.  This case is for a situation where you want to have two different versions of an app and serve them at the same time to do some usability or feature testing.